### PR TITLE
Add https://meahackclub.vercel.app/

### PR DESCRIPTION
**Pull request by shzcuber**

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -614,6 +614,10 @@ mcneil:
 - ttl: 1
   type: A
   value: 70.123.47.120
+mea:
```
Comment (sampoder) [inline hackclub.com.yaml:617]: Hey! Vercel doesn't allow for this style of CNAME. Furthermore, unfortunately due to platform restrictions we can't add non-HQ sites on Vercel to the DNS. I recommend using Netlify as an alternative. Sorry :)

**Review by sampoder — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (MEAHackClub) [reply hackclub.com.yaml:617]: Deployed to Netlify, please let me know if this works :p

**Review by sampoder — APPROVED** [review]

**Comment by MEAHackClub** [comment]
Hey, it doesn't seem like it's up yet. Should I give it a few days?

**Comment by sampoder** [comment]
Did you configure it on Netlify's end

**Comment by MEAHackClub** [comment]
Hmm, it seems to work fine: 
https://meahackclub.netlify.app/
What did I need to configure?

**Comment by sampoder** [comment]
The domains in netlify

**Comment by MEAHackClub** [comment]
ohhhhhhhhh i see i'll do it rn thanks

**Comment by sampoder** [comment]
coolio

**Comment by MEAHackClub** [comment]
image
Ah darn, am i doing something wrong?

**Comment by MEAHackClub** [comment]
image

**Comment by sampoder** [comment]
click Check DNS Config

**Comment by MEAHackClub** [comment]
image

File: hackclub.com.yaml [hunk 2]
```diff
@@ -614,6 +614,10 @@ mcneil:
...
+- ttl: 1
+  type: CNAME
+  value: https://meahackclub.netlify.app.
```
Comment (sampoder) [inline hackclub.com.yaml:620]: this shouldn't have https

**Comment by sampoder** [comment]
I believe I have fixed this

**Comment by MEAHackClub** [comment]
tyty! my bad i really should've caught that lolll

**Pull request merged**